### PR TITLE
Add link from QMM page to items catalog and levels

### PR DESCRIPTION
**Pull request by emranbm**

### CODE REVIEW

File: MATURITY_MODEL.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # QMM, in detail
-Qualitic Maturity Model consists of 5 levels; the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.
+Qualitic Maturity Model consists of [5 levels](/LEVELS.html); the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.  
+Each level is a list of maturity items, grouped together. [Maturity items](/CATALOG.html) are the building blocks of QMM; each of which indicates a specific hint about what should be performed, for the sake of quality.
```
Comment (mojyou) [inline MATURITY_MODEL.md:3]: I suggest:
Each level includes a list of ...

File: MATURITY_MODEL.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # QMM, in detail
-Qualitic Maturity Model consists of 5 levels; the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.
+Qualitic Maturity Model consists of [5 levels](/LEVELS.html); the first level introduces the initial steps to get in the picture of quality; whereas level 5 indicates the ultimate state of quality.
```
Comment (mojyou) [inline MATURITY_MODEL.md:2]: in the pull request #58, ".html" has been removed in levels.html address. does this relate to that or this is my mistake?
By the way, I mean please be careful on this.
Thanks.

**Review by mojyou — COMMENTED** [review]
Please have a look at my comments.
Thanks

File: MATURITY_MODEL.md (hunk 2 above)
Comment (emranbm) [reply MATURITY_MODEL.md:2]: You're right. I'm gonna fix it.

Comment (emranbm) [reply MATURITY_MODEL.md]: Done.

**Review by mojyou — APPROVED** [review]
Thanks Emran.

**Pull request merged**